src/components/connected/permanence/PermanencePrincipale.js

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { permanenceActions } from '../../../actions';
+
+import ListPermanences from './ListPermanences';
+import Recapitulatif from './Recapitulatif';
+import Horaires from './Horaires';
+import Adresse from './Adresse';
+import TypeAcces from './TypeAcces';
+
+function PermanencePrincipale({ permanence, structure }) {
+  const dispatch = useDispatch();
+
+  const adresseStructure = structure?.insee.etablissement.adresse;
+  const siretStructure = permanence?.siret ? String(permanence?.siret) : structure?.siret;
```

Comment:
```suggestion
  const siretStructure =  `${permanence?.siret ?? structure?.siret }`;
```

Devrait faire l'affaire